### ISSUE TITLE
-aa generates invalid boolean experession

### ISSUE THREAD

**Issue by kalekseev** [hhatto/autopep8#332]
```
$ autopep8 --version
autopep8 1.3.2 (pycodestyle: 2.0.0)
$ cat a.py
not a is None and not b is None
$ autopep8 -aa a.py
a is None and b is not not None
```

**Comment by spaceone** [comment]
See also https://github.com/hhatto/autopep8/issues/276

**Comment by hhatto** [comment]
thank you for reporting.

fix in #390.
autopep8 version 1.3.5 has been released.

Thanks.
